### PR TITLE
docs: add note about fuses to our security documentation

### PR DESCRIPTION
**Pull request by Kilian**
#### Description of Change

This adds a new item to the security checklist to review the fuses values. IT also adds item number 18 to the ToC up top.

#### Checklist

- [ ] PR description included and stakeholders cc'd
- [ ] `npm test` passes
- [ ] tests are [changed or added](https://github.com/electron/electron/blob/main/docs/development/testing.md)
- [x] relevant documentation, tutorials, templates and examples are changed or added
- [x] [PR release notes](https://github.com/electron/clerk/blob/main/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/main/README.md#examples).

#### Release Notes

Notes: none

### CODE REVIEW

File: docs/tutorial/security.md [hunk 1]
```diff
@@ -114,6 +114,8 @@ You should at least follow these steps to improve the security of your applicati
 15. [Do not use `shell.openExternal` with untrusted content](#15-do-not-use-shellopenexternal-with-untrusted-content)
 16. [Use a current version of Electron](#16-use-a-current-version-of-electron)
 17. [Validate the `sender` of all IPC messages](#17-validate-the-sender-of-all-ipc-messages)
+18. [Avoid usage of the `file://` protocol and prefer usage of custom protocols](#18-avoid-usage-of-the-file-protocol-and-prefer-usage-of-custom-protocols)
```
Comment (felixrieseberg) [inline docs/tutorial/security.md:117]: Good catch!

File: docs/tutorial/security.md [hunk 2]
```diff
@@ -789,3 +791,25 @@ learn how to serve files / content from a custom protocol.
...
+
+### 19. Check which fuses you can change
+
+Electron ships with a number of options that can be useful but a large portion of
+applications probably don't need. In order to avoid having to build your own version of
+Electron, these can be turned off or on using [Fuses](./fuses.md).
+
+#### Why?
+
+Some fuses, like `runAsNode` and `nodeCliInspect` allow the application to behave differently
```
Comment (felixrieseberg) [inline docs/tutorial/security.md:803]:
```suggestion
Some fuses, like `runAsNode` and `nodeCliInspect`, allow the application to behave differently
```

**Review by felixrieseberg — APPROVED** [review]

**Review by erickzhao — APPROVED** [review]

**Review by erickzhao — APPROVED** [review]
docs: lgtm

**Review by dsanders11 — CHANGES_REQUESTED** [review]
Can we move the new section above the reference link definitions so they stay at the bottom of the doc?

**Comment by Kilian** [comment]
@dsanders11 change made!

**Review by dsanders11 — APPROVED** [review]

**Pull request merged**